Update README.md

CescaGarrett opened the pull request:
Update language to align with the CHT glossary

abbyad commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # Medic Mobile
 
-These instructions should help you get setup to run or develop on Medic Mobile's Community Health Application Framework. For latest changes and release announcements see our [release notes](https://github.com/medic/medic/tree/master/release-notes).
+These instructions should help you get setup to run or develop on Medic Mobile's Core Framework. For latest changes and release announcements see our [release notes](https://github.com/medic/medic/tree/master/release-notes).
```
I think the "Core Framework" is in relation to the CHT, so perhaps something like this:
```suggestion
These instructions should help you get setup to run or develop on the Community Health Toolkit's Core Framework. For latest changes and release announcements see our [release notes](https://github.com/medic/medic/tree/master/release-notes).
```

Making this change may require other changes in the README though.

CescaGarrett commented:
#5827 Update language for accuracy and to explain our role within the Community Health Toolkit 

* removed reference to sim apps
* removed references to the apps as "medic"

CescaGarrett commented:
@abbyad Overview added! And then a TOC. I am clearly out of control.

abbyad commented on README.md:
```diff
@@ -1,37 +1,43 @@
+Mobile and web applications built with the Core Framework support a team-based approach to healthcare delivery and management. Health workers can use SMS messages or mobile applications to submit health data that can then be viewed and exported using a web application. These web applications are fully responsive with a mobile-first design, and support localization using any written language. They ...
 
-Currently, we functionally support the latest versions of Chrome, Chrome for Android and Firefox. We do not support Safari (unreliable implementations of web APIs we need) and the generic android browser (unreliable implementations in general). Our webapp code, which includes any code written as configuration, is still ES5. Our exact support matrix (including older app versions) can be found [in  ...
+Currently, the latest versions of Chrome, Chrome for Android and Firefox are functionally supported. We do not support Safari (unreliable implementations of necessary web APIs) and the generic android browser (unreliable implementations in general). Our webapp code, which includes any code written as configuration, is still ES5. Our exact support matrix (including older app versions) can be found ...
```
Might be nice to put this in a new section, like "Supported Browsers", but ok to live here for now.

CescaGarrett replied:
I agree! I could create "Supported Browsers" and put it after Supported Operating Systems @abbyad ?

CescaGarrett commented on README.md:
```diff
@@ -1,43 +1,49 @@
-We don't support Windows out of the box. However, you can try using the Windows Subsystem for Linux. See the [Windows Subsystem for Linux notes](https://github.com/medic/medic-docs/blob/master/development/using-windows.md) for how the installation instructions differ.
+### Supported Browsers
+
+Currently, the latest versions of Chrome, Chrome for Android and Firefox are functionally supported. We do not support Safari (unreliable implementations of necessary web APIs) and the generic android browser (unreliable implementations in general). Our webapp code, which includes any code written as configuration, is still ES5. Our exact support matrix (including older app versions) can be found ...
```
Full disclosure, I'm not sure if this line is actually about a supported browser. I kind of suspect it isn't: 

> Our webapp code, which includes any code written as configuration, is still ES5. Our exact support matrix (including older app versions) can be found [in our docs](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md).

abbyad replied:
The first quoted sentence is fine, but the second is about supported releases. It can fit in with this sentence in the intro:
> For the latest changes and release announcements see our [release notes](). **The support matrix of releases can be found in [our docs]().**

CescaGarrett replied:
I've moved it there!

abbyad replied:
Looks good!

CescaGarrett commented:
@SCdF or @garethbowen  Are you able to merge this for me? ✨
